.wordlist-en-custom.txt

```diff
@@ -0,0 +1,13 @@
+BigAnimal
+CIDR
+GUC
+IOPS
+InstanceType
+Postgres
+biganimal
```

Comment:
should Biganimal be capitalized?

Reply:
Unfortunately, spellchecker is not clever enough to distinguish when a word appears. This `biganimal` is coming from 
```
terraform {
  required_providers {
    biganimal = {
      source  = "biganimal"
      version = "0.3.1"
    }
  }
}
```

And, with the current implementation, if we use `biganimal` in, let's say, README.md, it will accept it. 🤷 not the best implementation, but I don't have any suggestions for fixing it. If you have, I'd love to hear. 

And I've only added the words that the spellchecker was complaining about. `Biganimal` wasn't one of them.